src/game/client/gui/gamewindowtransitions.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file
+ *
+ * @author tomsons26
+ *
+ * @brief Classes for handling WND UI system transitions.
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "gamewindowtransitions.h"
+
+void GameWindowTransitionsHandler::Init()
+{
+    m_group1 = nullptr;
+    m_group2 = nullptr;
+    m_group3 = nullptr;
+    m_group4 = nullptr;
+}
+
+void GameWindowTransitionsHandler::Reset()
+{
+    m_group1 = nullptr;
+    m_group2 = nullptr;
+    m_group3 = nullptr;
+    m_group4 = nullptr;
```

Comment:
What about `m_groupList` on Reset?

Reply:
it doesn't do anything with it but added a todo to invesigate